opentelemetry-user-events-metrics/src/exporter/mod.rs

```diff
@@ -0,0 +1,96 @@
+use crate::transform::transform_resource_metrics;
+use async_trait::async_trait;
+
+use opentelemetry_api::metrics::{MetricsError, Result};
+use opentelemetry_sdk::metrics::{
+    data::{ResourceMetrics, Temporality},
+    exporter::PushMetricsExporter,
+    reader::{AggregationSelector, TemporalitySelector},
+    Aggregation, InstrumentKind,
+};
+
+use crate::tracepoint;
+use prost::Message;
+use std::fmt::{Debug, Formatter};
+
+#[derive(Clone, Copy)]
+pub struct MetricsExporter {}
+
+impl MetricsExporter {
+    pub fn new() -> MetricsExporter {
+        // This is unsafe because if the code is used in a shared object (DLL),
```

Comment:
nit - remove DLL as it becomes windows specific. It is also unsafe in *inux  for dynamically loaded shared objects.